docs/source/development/documentation/api_generation.md

```diff
@@ -0,0 +1,56 @@
+---
+jupytext:
+  cell_metadata_filter: -all
+  formats: md:myst
+  main_language: python
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.13.8
+---
+
+# Rendered HTML for docstring example
+
+````{admonition} Rendering docstrings in the project web pages
+This page shows how the docstring example in our 
+[example docstring sample](docstring_style.md) page can be rendered in the project web 
+pages using the `sphinx` 
+[`autodoc`](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html)
+extension. 
+
+The docstring content of a module is converted to HTML by including a small Markdown
+file in the `docs/source/api` folder. Typically, the file will start with the
+`jupytext` YAML metadata, then a markdown title and then an `automodule` instruction to
```

Comment:
Feel like it might be worth spending a bit more time what each of these are for/why they are necessary